### ISSUE TITLE
Pylance scans libraries for problems repeatedly

### ISSUE THREAD

**Issue by nopeless** [microsoft/pylance-release#4401]
I do not know why this happens but when I enable black formatter it constantly scans libraries and reports errors in problems

**Comment by nopeless** [comment]
```
 (8532) Found 1 source file
 (8532) Background analysis(1) root directory: c:\Users\nope\.vscode\extensions\ms-python.vscode-pylance-2023.5.30\dist
 ```
![Code_zZyD496fLh](https://github.com/microsoft/pylance-release/assets/38830903/accd5654-bd91-449e-a40b-5f490f589d12)
 I believe this is causing the issue

**Comment by heejaechang** [comment]
@nopeless Can you provide us some logs as described in [the troubleshooting guide](https://github.com/microsoft/pylance-release/blob/main/TROUBLESHOOTING.md#filing-an-issue)?

or you could provide us repro steps.

**Comment by nopeless** [comment]
@heejaechang thanks for the interest. I looked into logs and they pointed to `%appdata%/Roaming/Python`. For now, I have just deleted that directory and reinstalled python (which seems to have "fixed" the issue). I am on my laptop and I don't have much time to look into this issue, as I rarely use python for the time being but I will reopen/link this issue if I encounter again. The original title was my misunderstanding as I have enabled both pylance and black at the same time, and I was under the impression that pylance would not be the cause of a major issue like this. 

TL;DR: I have fixed this issue via a hack and I hope this does not happen again.
